corehq/apps/users/analytics.py

```diff
@@ -82,8 +82,11 @@ def get_search_users_in_domain_es_query(domain, search_string, limit, offset):
     return user_es.start(offset).size(limit).sort('username.exact')
 
 
-def get_role_user_count(domain, role_id):
+def get_role_user_count(domain, role_id, web_users_only=True):
```

Comment:
Looks like this PR has deviated from https://github.com/dimagi/commcare-hq/pull/34544. Specifically, it needs https://github.com/dimagi/commcare-hq/pull/34544/commits/494546b934dd9c3ae90c085bf425c85a7a27e092

Reply:
thanks, updated